components/camel-plc4x/src/main/java/org/apache/camel/component/plc4x/Plc4XConsumer.java

```diff
@@ -85,6 +86,15 @@ protected void doStart() throws Exception {
     }
 
     private void startUnTriggered() {
+        if (plc4XEndpoint.isAutoReconnect() && !plcConnection.isConnected()) {
+            try {
+                plc4XEndpoint.reconnect();
+                LOGGER.debug("Successfully reconnected");
+            } catch (PlcConnectionException e) {
+                LOGGER.error("Unable to reconnect, skipping request", e);
```

Comment:
Maybe tone down the logging to WARN as you are dealing with this error by skipping.